src/pages/Inventory.page.js

```diff
@@ -29,6 +31,10 @@ export class InventoryPage extends BaseSwagLabPage {
         await this.addItemToCartBtns.nth(id).click();
     }
 
+    async getInventoryItems() {
+        return this.inventoryItems.count();
```

Comment:
1. No sence in this function, we just replaced `inventoryItems.count()` call with `getInventoryItems()` Both are 1 liners and this wrapper adds no logic or readability.
2. Method name `getInventoryItems` tells us, we will get items, but we get count.
3. no sence in async declaration since we dont use await